### PR TITLE
Added myself to contributors.md file after book dash 2022

### PR DESCRIPTION
**Pull request by RayStick**
### Summary

* Responds to #2723

* After joining the Nov 2022 BookDash event, I added myself to the contributor.md file!

### List of changes proposed in this PR (pull-request)

* I added my name to the contributor file and described my highlights of bookdash

### What should a reviewer concentrate their feedback on?

- [ ] I only changed line 1014 but strangely it looks like lots of lines have been removed then added - do you know why that happened?
- [ ] When I look at 'Deploy Preview' I don't actually see my change - am I meant to?
- [ ] Check formatting is correct and I am in line with general contributing guidelines (as this is my first PR 😄)
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [X] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:   @RayStick

### CODE REVIEW

**Comment by RayStick** [comment]
### Questions

-  I only changed line 1014 but strangely it looks like lots of lines have been removed then added - do you why that happened?
- When I look at 'Deploy Preview' above I don't actually see my change - am I meant to?

File: contributors.md [hunk 1]
```diff
@@ -1011,6 +1011,18 @@ I am also an associate member of the Chartered Institute of Personnel and Develo
...
+
+* Personal highlights:
+> Joining the Nov 2022 BookDash was a real pleasure - I met lots of great people with brilliant ideas, and I enjoyed learning more about The Turing Way. I am new to this community but felt welcomed and included. Working with the Scriberia artist on visualising the 'Data Wrangler' job role was a really interesting and useful experience, and it encouraged our wider team, based at The Alan Turing Institute, to reflect on our roles as Data Wranglers and how to explain this role to others. My contribution to the book was working with my colleague Mahwish Mohammad on adding a sub-chapter 'Data Wranglers' to the [Research Infrastructure Roles](https://the-turing-way.netlify.app/collaboration/research-infrastructure-roles.html) section, and starting (or joining) discussions with others in this community about Research Infrastructure Roles in general. I hope more collaborations will come out of these discussions, specifically content added to the book which compares and contrasts different Research Infrastructure Roles, and reflects on how to decide which role(s) would be needed for a specific research project.
```
Comment (aleesteele) [inline contributors.md:1024]: Looks great! Just adding a small edit to italicise The Turing Way, as it's how we usually cite/write about the book.
```suggestion
> Joining the Nov 2022 BookDash was a real pleasure - I met lots of great people with brilliant ideas, and I enjoyed learning more about _The Turing Way_. I am new to this community but felt welcomed and included. Working with the Scriberia artist on visualising the 'Data Wrangler' job role was a really interesting and useful experience, and it encouraged our wider team, based at The Alan Turing Institute, to reflect on our roles as Data Wranglers and how to explain this role to others. My contribution to the book was working with my colleague Mahwish Mohammad on adding a sub-chapter 'Data Wranglers' to the [Research Infrastructure Roles](https://the-turing-way.netlify.app/collaboration/research-infrastructure-roles.html) section, and starting (or joining) discussions with others in this community about Research Infrastructure Roles in general. I hope more collaborations will come out of these discussions, specifically content added to the book which compares and contrasts different Research Infrastructure Roles, and reflects on how to decide which role(s) would be needed for a specific research project.
```

**Review by aleesteele — APPROVED** [review]

**Review by aleesteele — APPROVED** [review]

**Comment by RayStick** [comment]
Thanks @aleesteele! Two questions if you don't mind (for my own understanding) - 

- I only changed line 1014 but strangely it looks like lots of lines have been removed then added - do you know why that happened?
- When I look at 'Deploy Preview' I don't actually see my change - am I meant to?

**Review by da5nsy — APPROVED** [review]

**Comment by RayStick** [comment]
@da5nsy, thanks for explaining, that is all clear now!

**Pull request merged**